backend/app/testbot/data_loader.py

```
@@ -95,16 +278,235 @@ def load_excel(file_path: str) -> List[str]:
         
         return chunks
     except Exception as e:
         logger.error(f"Error loading Excel: {e}")
+        return []
+
+def load_json(file_path: str) -> List[Dict[str, Any]]:
+    """
+    Load JSON files and convert to a list of dictionaries
+    """
+    try:
+        with open(file_path, 'r', encoding='utf-8') as f:
```

Ruff (0.8.2)

Unnecessary open mode parameters

Remove open mode parameters

(UP015)